### PR TITLE
ac4d bug fix

### PR DESCRIPTION
**Pull request by AbigailTCameron**

### CODE REVIEW

File: contracts/strategies/strategy-axial-4pool-base.sol [hunk 1]
```diff
@@ -120,23 +122,24 @@ abstract contract StrategyAxial4PoolBase is StrategyAxialBase {
...
             uint256 _keep = _axial.mul(keep).div(keepMax);
-            uint256 _amount = _axial.sub(_keep);
+    
             if (_keep > 0) {
                 _takeFeeAxialToSnob(_keep);
             }
-
+            
+            _axial = IERC20(axial).balanceOf(address(this));
             // if the stablecoin we need is frax, then swap with pangolin
```
Comment (saito-sv) [inline contracts/strategies/strategy-axial-4pool-base.sol:131]: this comment is no longer valid.

Comment (AbigailTCameron) [reply contracts/strategies/strategy-axial-4pool-base.sol]: Good catch! Updated

**Review by saito-sv — APPROVED** [review]
LGTM

**Comment by AbigailTCameron** [comment]
waiting on fuzzylemma's changes to update controller

**Comment by fuzzylemma** [comment]
> waiting on fuzzylemma's changes to update controller

I satisfied merge requirements. Although, we should get jomaris [PR144](https://github.com/Snowball-Finance/protocol/pull/144) merged first. This contains files master is missing causing tests to fail. Which i can then update the type script with.

**Comment by AbigailTCameron** [comment]
New PR with same logic

**Pull request closed without merging**